source/styleEditor.js

```diff
@@ -1,3 +1,14 @@
+const processSubmit = (e) => {
```

Comment:
If you're into it, you can drop the parens on single argument arrow functions. A lot of "authoritative" ES6 style guides (e.g., Airbnb) prefer it.